src/Compilers/CSharp/Portable/Symbols/Metadata/PE/SymbolFactory.cs

```diff
@@ -57,6 +57,11 @@ internal override bool IsVolatileModifierType(PEModuleSymbol moduleSymbol, TypeS
             return type.SpecialType == SpecialType.System_Runtime_CompilerServices_IsVolatile;
         }
 
+        internal override bool IsConstModifierType(PEModuleSymbol moduleSymbol, TypeSymbol type)
+        {
+            return type.ToDisplayString() == WellKnownTypes.GetMetadataName(WellKnownType.System_Runtime_CompilerServices_IsConst);
+        }
```

Comment:
If going with name comparing, should definitely compare namespace/type name parts separately. I think ToDisplay will concat them every time

---
In reply to: [118063143](https://github.com/dotnet/roslyn/pull/19658#discussion_r118063143) [](ancestors = 118063143)